### PR TITLE
first navbar responsive iteration

### PR DESCRIPTION
**Pull request by mfc-boost**
added support via CSS media-queries to get responsive navbar behaviour

### CODE REVIEW

File: components/MenuMobile/index.js [hunk 1]
```diff
@@ -7,23 +7,15 @@ import Link from 'next/link';
...
+const Menu = ({ router, links, desktop }) => (
   <div className="wrapper">
     <input type="checkbox" id="menuToggler" className="input-toggler" />
     <label htmlFor="menuToggler" className="menu-toggler">
       <span className="menu-toggler__line" />
       <span className="menu-toggler__line" />
       <span className="menu-toggler__line" />
     </label>
-    <aside className="sidebar">
+    <aside className={`sidebar ${desktop && 'desktop'}`}>
```
Comment (fc-boost) [inline components/MenuMobile/index.js]: use classnames

File: components/MenuMobile/index.js [hunk 2]
```diff
@@ -134,6 +126,11 @@ const Menu = ({ router }) => (
         transition: transform 0.6s, background-position 1s 0.6s;
       }
 
+      .desktop {
```
Comment (fc-boost) [inline components/MenuMobile/index.js]: `.component--desktop`?

File: components/NavBar.js [hunk 3]
```diff
@@ -27,6 +28,9 @@ const NavBar = ({ router }) => {
             <Logo inverted />
           </a>
         </div>
+        <span className="menu">
```
Comment (fc-boost) [inline components/NavBar.js]: Always prefer `div` over `span` to prevent HTML nesting errors

File: components/NavBar.js [hunk 4]
```diff
@@ -125,6 +137,15 @@ const NavBar = ({ router }) => {
           width: 44px; /* added width to allow IE 10+ to correctly render the SVG */
           padding: 0 ${paddingBase};
         }
+
+        @media (max-width: 1025px) {
```
Comment (fc-boost) [inline components/NavBar.js]: Move viewport sizes into the styles.js

File: components/PageLayout/index.js [hunk 5]
```diff
@@ -21,13 +21,18 @@ const PageLayout = ({ title, isHome, isExp, children }) => {
...
+        {isHome ? <Intro /> : null}
+        <div className={pageLayoutClasses}>
+          {children}
+          <GetInTouch />
+        </div>
       </div>
       <Footer />
       <style jsx>{`
+        .page-content {
+          margin-top: 100px;
```
Comment (fc-boost) [inline components/PageLayout/index.js]: Maybe we could move this value into styles.js to keep it in sync only in one place

File: components/NavBar.js (hunk 4 above)
Comment (mfc-boost) [reply components/NavBar.js]: Styled-JSX doesn't support properly evaluating string to define mq

**Pull request merged**